extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/validation/JPAValidationDisabledTestCase.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.hibernate.orm.validation;
+
+import static org.hamcrest.Matchers.is;
+
+import javax.transaction.Transactional;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.hibernate.orm.MyEntity;
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class JPAValidationDisabledTestCase {
+
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addClasses(MyEntity.class, JPATestValidationResource.class)
+                    .addAsResource("application-validation-disabled.properties", "application.properties"));
+
+    //define an empty import.sql file
+
+    @Test
+    @Transactional
+    public void testValidEntity() {
+        String entityName = MyEntity.ENTITY_NAME_TOO_LONG;
```

Comment:
This constant is not a value that's too long to fit the entity name, but the message displayed when an entity name is too long.

To actually test that validation is disabled, see how the validation failure is triggered other tests:

```suggestion
        String entityName = "Post method should not persist an entity having a Size constraint of 50 on the name column.";
```